<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>moyenmain</class>
 <widget class="QMainWindow" name="moyenmain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow
{
background-color:#131E3A;
}
QLabel
{
background-color:black;
font-size:20px
}
QPushButton:hover
{
color:white;
border-radius:15px;
background:#000080;
}







QToolTip
{
     border: 1px solid black;
     background-color: #ffa02f;
     padding: 1px;
     border-radius: 3px;
     opacity: 100;
}

QWidget
{
    color: black;
    background-color: #323232;
}

QWidget:item:hover
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #ca0619);
    color: #000000;
}

QWidget:item:selected
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}

QMenuBar::item
{
    background: transparent;
}

QMenuBar::item:selected
{
    background: transparent;
    border: 1px solid #ffaa00;
}

QMenuBar::item:pressed
{
    background: #444;
    border: 1px solid #000;
    background-color: QLinearGradient(
        x1:0, y1:0,
        x2:0, y2:1,
        stop:1 #212121,
        stop:0.4 #343434/*,
        stop:0.2 #343434,
        stop:0.1 #ffaa00*/
    );
    margin-bottom:-1px;
    padding-bottom:1px;
}

QMenu
{
    border: 1px solid #000;
}

QMenu::item
{
    padding: 2px 20px 2px 20px;
}

QMenu::item:selected
{
    color: #000000;
}

QWidget:disabled
{
    color: #404040;
    background-color: #323232;
}

QAbstractItemView
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0.1 #646464, stop: 1 #5d5d5d);
}

QWidget:focus
{
    /*border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);*/
}

QLineEdit
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #000080, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid blue;
    border-radius: 5;
}

QPushButton
{
    color: black;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #0000A0, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    font-size: 22px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

QComboBox
{
    selection-background-color: #ffaa00;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}

QComboBox:hover,QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}


QComboBox:on
{
    padding-top: 3px;
    padding-left: 4px;
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
    selection-background-color: #ffaa00;
}

QComboBox QAbstractItemView
{
    border: 2px solid darkgray;
    selection-background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}

QComboBox::drop-down
{
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;

     border-left-width: 0px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }

QComboBox::down-arrow
{
     image: url(:/down_arrow.png);
}

QGroupBox:focus
{
border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}

QTextEdit:focus
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 black, stop: 1 black);
}

QScrollBar:horizontal {
     border: 1px solid #222222;
     background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0.0 #121212, stop: 0.2 #282828, stop: 1 #484848);
     height: 30px;
     margin: 0px 16px 0 16px;
}

QScrollBar::handle:horizontal
{
      background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 #ffa02f, stop: 0.5 #d7801a, stop: 1 #ffa02f);
      min-height: 20px;
      border-radius: 2px;
}

QScrollBar::add-line:horizontal {
      border: 1px solid #1b1b19;
      border-radius: 2px;
      background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 #ffa02f, stop: 1 #d7801a);
      width: 20px;
      subcontrol-position: right;
      subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
      border: 1px solid #1b1b19;
      border-radius: 2px;
      background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 #ffa02f, stop: 1 #d7801a);
      width: 20px;
     subcontrol-position: left;
     subcontrol-origin: margin;
}

QScrollBar::right-arrow:horizontal, QScrollBar::left-arrow:horizontal
{
      border: 1px solid black;
      width: 1px;
      height: 1px;
      background: white;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
      background: none;
}

QScrollBar:vertical
{
      background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 0.0 #121212, stop: 0.2 #282828, stop: 1 #484848);
      width: 7px;
      margin: 16px 0 16px 0;
      border: 1px solid #222222;
}

QScrollBar::handle:vertical
{
      background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 0.5 #d7801a, stop: 1 #ffa02f);
      min-height: 20px;
      border-radius: 2px;
}

QScrollBar::add-line:vertical
{
      border: 1px solid #1b1b19;
      border-radius: 2px;
      background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
      height: 14px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical
{
      border: 1px solid #1b1b19;
      border-radius: 2px;
      background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #d7801a, stop: 1 #ffa02f);
      height: 14px;
      subcontrol-position: top;
      subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
{
      border: 1px solid black;
      width: 1px;
      height: 1px;
      background: white;
}


QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
      background: none;
}

QTextEdit
{
    color: black;
}

QPlainTextEdit
{
    background-color: #000080;
}

QHeaderView::section
{
    background-color: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop:0 #616161, stop: 0.5 #505050, stop: 0.6 #434343, stop:1 #656565);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
font-size:20px;
}

QCheckBox:disabled
{
color: #414141;
}

QDockWidget::title
{
    text-align: center;
    spacing: 3px; /* spacing between items in the tool bar */
    background-color: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop:0 #323232, stop: 0.5 #242424, stop:1 #323232);
}

QDockWidget::close-button, QDockWidget::float-button
{
    text-align: center;
    spacing: 1px; /* spacing between items in the tool bar */
    background-color: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop:0 #323232, stop: 0.5 #242424, stop:1 #323232);
}

QDockWidget::close-button:hover, QDockWidget::float-button:hover
{
    background: #242424;
}

QDockWidget::close-button:pressed, QDockWidget::float-button:pressed
{
    padding: 1px -1px -1px 1px;
}

QMainWindow::separator
{
    background-color: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop:0 #161616, stop: 0.5 #151515, stop: 0.6 #212121, stop:1 #343434);
    color: white;
    padding-left: 4px;
    border: 1px solid #4c4c4c;
    spacing: 3px; /* spacing between items in the tool bar */
}

QMainWindow::separator:hover
{

    background-color: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop:0 #d7801a, stop:0.5 #b56c17 stop:1 #ffa02f);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
    spacing: 3px; /* spacing between items in the tool bar */
}

QToolBar::handle
{
     spacing: 3px; /* spacing between items in the tool bar */
     background: url(:/images/handle.png);
}

QMenu::separator
{
    height: 2px;
    background-color: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop:0 #161616, stop: 0.5 #151515, stop: 0.6 #212121, stop:1 #343434);
    color: white;
    padding-left: 4px;
    margin-left: 10px;
    margin-right: 5px;
}

QProgressBar
{
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
}

QProgressBar::chunk
{
    background-color: #d7801a;
    width: 2.15px;
    margin: 0.5px;
}



QRadioButton::indicator:checked, QRadioButton::indicator:unchecked{
    color: #b1b1b1;
    background-color: #323232;
    border: 1px solid #b1b1b1;
    border-radius: 6px;
}

QRadioButton::indicator:checked
{
    background-color: qradialgradient(
        cx: 0.5, cy: 0.5,
        fx: 0.5, fy: 0.5,
        radius: 1.0,
        stop: 0.25 #ffaa00,
        stop: 0.3 #323232
    );
}

QCheckBox::indicator{
    color: #b1b1b1;
    background-color: #323232;
    border: 1px solid #b1b1b1;
    width: 9px;
    height: 9px;
}

QRadioButton::indicator
{
    border-radius: 6px;
}

QRadioButton::indicator:hover, QCheckBox::indicator:hover
{
    border: 1px solid #ffaa00;
}

QCheckBox::indicator:checked
{
    image:url(:/images/checkbox.png);
}

QCheckBox::indicator:disabled, QRadioButton::indicator:disabled
{
    border: 1px solid #444;
}

QTabBar::tab {
    color: white;
    border: 1px solid #444;
    border-bottom-style: none;
    background-color: black;
    padding-left: 10px;
    padding-right: 10px;
    padding-top: 5px;
    padding-bottom: 2px;
    margin-right: -1px;
}

QTabWidget::pane {
    border: 1px solid #444;
    top: 1px;
}

QTabBar::tab:last
{
    margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
    border-top-right-radius: 3px;
}

QTabBar::tab:first:!selected
{
 margin-left: 0px; /* the last selected tab has nothing to overlap with on the right */


    border-top-left-radius: 3px;
}

QTabBar::tab:!selected
{
    color: #b1b1b1;
    border-bottom-style: solid;
    margin-top: 3px;
    background-color: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop:1 #212121, stop:.4 #343434);
}

QTabBar::tab:selected
{
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    margin-bottom: 0px;
}

QTabBar::tab:!selected:hover
{
    /*border-top: 2px solid #ffaa00;
    padding-bottom: 3px;*/
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    background-color: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop:1 #212121, stop:0.4 #343434, stop:0.2 #343434, stop:0.1 #ffaa00);
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>871</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>181</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 85, 127);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="moyens">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Moyen URbain</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reservations">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Moyen Rurale</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>GESTIONS</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>691</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 85, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>SMART_TRANSPORT</string>
       </property>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>651</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover
{
color:white;
border-radius:15px;
background:#000080;
}</string>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page_3"/>
      <widget class="QWidget" name="page_4"/>
      <widget class="QWidget" name="page_5">
       <widget class="QTabWidget" name="moyen1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>621</width>
          <height>391</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_12">
         <attribute name="title">
          <string>Ajouter/Modifier</string>
         </attribute>
         <widget class="QLineEdit" name="lineEdit_numeroM2">
          <property name="geometry">
           <rect>
            <x>371</x>
            <y>10</y>
            <width>231</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Chercher le numero à  supprimer</string>
          </property>
         </widget>
         <widget class="QPushButton" name="trier1">
          <property name="geometry">
           <rect>
            <x>503</x>
            <y>60</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>trier</string>
          </property>
         </widget>
         <widget class="QPushButton" name="supprimer1">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>120</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>231</width>
            <height>343</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_49">
             <item>
              <widget class="QLabel" name="label_50">
               <property name="text">
                <string>Numero</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_numeroM"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_50">
             <item>
              <widget class="QLabel" name="label_51">
               <property name="text">
                <string>type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_typeM"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_51">
             <item>
              <widget class="QLabel" name="label_52">
               <property name="text">
                <string>etat</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_etatM"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="ajouter1">
             <property name="text">
              <string>Ajouter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="modifier1">
             <property name="text">
              <string>Modifier</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_11">
         <attribute name="title">
          <string>Moyens</string>
         </attribute>
         <widget class="QTableView" name="typemoyen1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>601</width>
            <height>351</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_15">
         <attribute name="title">
          <string>Rechercher</string>
         </attribute>
         <widget class="QLineEdit" name="chermoyen">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>501</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Entrer le numero à rechercher</string>
          </property>
         </widget>
         <widget class="QTableView" name="chermoyen_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>601</width>
            <height>301</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_11">
       <widget class="QTabWidget" name="moyen2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>621</width>
          <height>401</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_19">
         <attribute name="title">
          <string>Ajouter/Modifier</string>
         </attribute>
         <widget class="QLineEdit" name="lineEdit_numeroM12">
          <property name="geometry">
           <rect>
            <x>371</x>
            <y>10</y>
            <width>231</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Chercher le numero à  supprimer</string>
          </property>
         </widget>
         <widget class="QPushButton" name="trier2">
          <property name="geometry">
           <rect>
            <x>503</x>
            <y>60</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>trier</string>
          </property>
         </widget>
         <widget class="QPushButton" name="supprimer2">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>120</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>231</width>
            <height>343</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_52">
             <item>
              <widget class="QLabel" name="label_53">
               <property name="text">
                <string>Numero</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_numeroM1"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_53">
             <item>
              <widget class="QLabel" name="label_54">
               <property name="text">
                <string>type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_typeM1"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_54">
             <item>
              <widget class="QLabel" name="label_55">
               <property name="text">
                <string>etat</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_etatM1"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="ajouter2">
             <property name="text">
              <string>Ajouter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="modifier2">
             <property name="text">
              <string>Modifier</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_23">
         <attribute name="title">
          <string>Moyens</string>
         </attribute>
         <widget class="QTableView" name="typemoyen2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>601</width>
            <height>351</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_24">
         <attribute name="title">
          <string>Rechercher</string>
         </attribute>
         <widget class="QLineEdit" name="chermoyen_5">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>501</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Entrer le numero à rechercher</string>
          </property>
         </widget>
         <widget class="QTableView" name="chermoyen_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>601</width>
            <height>311</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6"/>
      <widget class="QWidget" name="page_7"/>
      <widget class="QWidget" name="page_8"/>
      <widget class="QWidget" name="page_9"/>
      <widget class="QWidget" name="page_10"/>
     </widget>
     <zorder>stackedWidget_2</zorder>
     <zorder>frame</zorder>
     <zorder>frame_2</zorder>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
